confidential_backend/secondary_fhir_strategy.py

```diff
@@ -0,0 +1,173 @@
+"""Source Strategy implementation for a FHIR server in a secondary (non launch) role."""
+import re
+import requests
+from urllib.parse import urlencode, urlparse, urlunparse, parse_qsl
+
+from flask import current_app, has_request_context
+
+from confidential_backend.wrapped_session import get_session_value, set_session_value
+from confidential_backend.source_strategy import SourceStrategy
+
+class SecondaryFhirStrategy(SourceStrategy):
```

Comment:
Could these strategy files be moved to a subdirectory? 

I can imagine we might have strategies from *-environments we might want to inject, without needing to add them to uwcirg/helloworld-confidential-client-sof